File: fizzbuzz.py
#!/usr/bin/python
limit = 1000
for count in range(0, limit + 1):
    fizz = count % 3
    buzz = count % 5
    msg = str(count)
    if fizz == 0:
        msg = msg + " Fizz"
    if buzz == 0:
        msg = msg + " Buzz"
    print(msg)

File: fizzbuzz.java
class fizzbuzz {
  public static void main(String[] args) {
    int limit = 1000;
    for(int count = 0; count <= limit; count++) {
      int fizz = count % 3;
      int buzz = count % 5;
      String msg = String.valueOf(count);
      if(fizz == 0) msg += " Fizz";
      if(buzz == 0) msg += " Buzz";
      System.out.println(msg);
    }
  }
}
